packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -0,0 +1,432 @@
+  "lastUpdatedBy": "00ufnlhzppWItClAI0g4",
+  "name": "aNewType",
+  "ref": {
+    "id": "otyfnly5cQjJT9PnR0g4",
+    "refClass": "com.saasure.db.dto.platform.entity.cvd.CVDType"
+  }
+}
+```
+
+### Delete User Type
+
+<ApiOperation method="delete" url="/api/v1/meta/types/user/${typeId}" />
+
+Deletes a User Type permanently. This operation is not permitted for the default type, nor if there are non-deleted users with the specified type. After a User Type has been deleted, it cannot be used as the type for new users, and it no longer counts against the limit of 10 User Types.
+
+##### Request Parameters
+
+| Parameter | Description                                                                           | Param Type | DataType | Required | Default |
+| --------- | ------------------------------------------------------------------------------------- | ---------- | -------- | -------- | ------- |
+| typeId    | `id` of user type                                                                     | URL        | String   | TRUE     |         |
+
+##### Response Parameters
+
+None.
+
+Passing an invalid `typeId` returns a `404 Not Found` status code with error code `E0000007`.
+
+Passing the id of the default type returns a `403 Forbidden` status code with error code `E0000142` and a `reason` of `PROHIBITED`.
+
+Passing the id of a type for which non-deleted users exist returns a `403 Forbidden` status code with error code `E0000142` and a `reason` of `UNMET_REQUIREMENTS`.
+
+##### Request Example
+
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/meta/types/user/otyfnly5cQjJT9PnR0g4"
+```
+
+##### Response Example
+
+
+```http
+HTTP/1.1 204 No Content
+```
+
+## Specify the User Type of a New User
+
+The [Create User](/docs/reference/api/users/#create-user-with-non-default-user-type) operation accepts a type specification as part of the request body. The specification is a map, but currently the only key permitted is "id". The type specification is also added to the [User Model](/docs/reference/api/users/#user-model) (see [User object updates](#user-object-updates)), but after user creation the type is read-only.
+
+##### Example
+
+```json
+  "type": {
+    "id": "otyfnjfba4ye7pgjB0g4"
+  }
+```
+
+## User Schema Operation Extensions to Non-Default Types
+
+All operations documented under [User Schema Operations](/docs/reference/api/schemas/#user-schema-operations) will continue to operate as before. All these APIs now have an alternate form - replace the trailing `/default` with a schema ID - which will perform the relevant operation on the schema associated with any type.
+
+Just as a schema is currently associated with a User Type, the schema ID is related to the type ID: replace the leading `oty` with `osc`. For example, a User Type with ID `oty1234567890abcdefg` would have a Schema ID of `osc1234567890abcdefg`.
```

Comment:
How about: "Just as a schema is currently associated with a User Type, a schema ID is related to a type ID, with the leading `oty` replaced by `osc`."